pkgs/os-specific/linux/tailor-gui/default.nix

```diff
@@ -17,7 +17,7 @@ let
   src = tuxedo-rs.src;
   sourceRoot = "source/tailor_gui";
   pname = "tailor_gui";
-  version = tuxedo-rs.version;
+  version = "0.2.3";
```

Comment:
inheriting source and then overriding version sounds fishy to me. We should either take both or fetch the src here.

Reply:
Do you see any actual pitfalls to this?

The crates in tuxedo-rs have versions that update independently, but the packages still belong together, and ideally the sources shouldn't be out of sync.

I would read the versions from the respective Cargo.toml, but that would be IFD...

Reply:
tuxedo-rs could be updated and also include an update to tuxedo-gui but we could forget to update the version number here.

Reply:
And if we use separate sources, tuxedo-rs could be updated and we'd forget to update tailor-gui, which would potentially break tailor-gui.
With shared sources, tailor-gui will fail to build if we update the source and the cargo hash changes.

This could potentially be solved with a `passthru.tests.version` for tailor-gui.
But as it stands, rust GTK apps don't support passing in a `--version` flag (afaik).
I've raised [an issue about this upstream](https://github.com/AaronErhardt/tuxedo-rs/issues/61). Maybe there's a way to do it.

Reply:
I think this is good enough for now. No reason to block the PR over this.

Reply:
Just don't forget to update this when r-ryantm comes by.